oemof/solph/components.py

```diff
@@ -358,7 +357,7 @@ def _storage_balance_first_rule(block, n):
             expr = 0
             expr += block.capacity[n, 0]
             expr += - block.init_cap[n] * (
-                1 - n.loss_rate[0])
+                (1 - n.loss_rate[0]) ** m.timeincrement[0])
```

Comment:
This changes the result w.r.t. the old version with constant timeincrement unless `m.timeincrement[t] == 1`. So, the default timeincrement (when not explicitly giving one) should be 1.

Reply:
That's the most difficult part. It depends on loss_rate. Is the given rate describing the loss per hour, 15 mins,...? My assumption is per hour, in documentation: "loss_rate: The relative loss of the storage capacity per timeunit." (you're right, I have to be more clear about the timeunit). In that case there is no need to set a default timeincrement. Please correct me if I'm wrong.

Reply:
It is not possible to define a default `timeincrement`, because it belongs to the time index!

Reply:
Before, the loss_rate (sequence) might have already included a varying time increment: Assuming 10% losses per hour with time steps of `[1 h, 2 h, 1 h]`, I would have passed `[0.1, 0.19, 0.1]`. This allowed us to not define the units. The change per step is already defined.

As we assumed equidistant time steps, timeunit is just the width of the time steps, exactly what would be `timeincrement == 1` in your case.

Reply:
If you're only considering the calculation above for a default timeincrement, then it might be possible to set a default `timeincrement == 1`, if and only if the user is really aware about calculating the loss_rate for each timestep. Wouldn't it be more user friendly, if the user doesn't have to calculate it all by hand?
Also, if you're considering all the other calculations, which include the timeincrement, it could lead to wrong calculations, e.g. for calculating the summed maximum of a flow:
```python
lhs = sum(m.flow[inp, out, ts] * m.timeincrement[ts]
                   for ts in m.TIMESTEPS)

rhs = (m.flows[inp, out].summed_max *
             m.flows[inp, out].nominal_value)
```

Reply:
This is totally correct. We should just add a comment to the release notes, that the loss rate is now defined per hour and no more per time step.